src/Products/PluginIndexes/cache.py

```diff
@@ -22,13 +20,11 @@ class RequestCache(dict):
     _sets = 0
 
     def get(self, key, default=None):
-        value = super(RequestCache, self).get(key, _marker)
-
-        if value is _marker:
-            self._misses += 1
+        try:
+            value = self[key]
+        except KeyError:
             return default
 
-        self._hits += 1
```

Comment:
I do not understand, why this method used to set `_misses` and `_hits`, but does not need it anymore. As it seems to be actually a tool for testing, it should be fine, if nothing breaks.

Reply:
The "get" method now calls the"__getitem__" method (see self[key]), which still sets the variables"_hits" and"_misses". Other tests using the method will therefore continue to work.